packages/jest-core/src/getProjectDisplayName.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {Config} from '@jest/types';
+
+export default function getProjectDisplayName(
```

Comment:
This looks like https://github.com/facebook/jest/blob/d7a7b4294a4507030f86fe4f78e1790f53d0bda9/packages/jest-reporters/src/utils.ts#L18-L34 without the color. Can we share the code somehow?

Thinking about it, `normalize` should probably spit out the object form with the default `white` color, so we'd only have to deal with the one form inside jest

Reply:
That's what I was telling myself when I wrote this. However this bit of code is in `jest-reporters` and I didn't if or where I should move the code.

Leveraging `normalize` makes more sense to me. I'll try that.

Reply:
I tried and I failed. I can't give an detailed account of the blockers because it was too long ago.

Reply:
Opened up #10010 for this btw, so we don't forget 🙂